script/generate-llms-full.mjs

```
@@ -254,6 +254,81 @@ function sidebarTopLevelSections(docs) {
+// `![alt](…svg)` markdown image); in the generated text reference a raw `<img>`
+// is dead weight — the relative path resolves to nothing and there is no visual
+// — so we surface the alt text instead. See describeSvgDiagrams for why code
+// blocks never reach this function.
+function rewriteDiagramEmbeds(text) {
+  return text
+    .replace(/<p\b[^>]*>\s*(<img\b[^>]*>)\s*<\/p>/gi, (whole, img) => {
+      const rewritten = rewriteImgTag(img);
+      return rewritten === img ? whole : rewritten;
+    })
+    .replace(/<img\b[^>]*>/gi, (img) => rewriteImgTag(img))
+    .replace(/!\[([^\]]*)\]\([^)\s]*\.svg\)/gi, (whole, alt) => diagramMarker(alt));
```

ESLint

[error] 'whole' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)